test/local/account/getBlockIntervalState.test.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from 'chai'
+import { getBlockIntervalState } from '@brink-sdk'
+
+describe('getBolckIntervalState()', function () {
+  it('when slot is not used, should return 0', function () {
+    const slot = 0
+    const { counter, start } = getBlockIntervalState({ slot })
+    expect(counter).to.equal(0)
+    expect(start).to.equal(slot)
+  })
+
+  it("when is all 1's, should return true", function () {
```

Comment:
could say `when slot is max uint256 value, should return uint16 max value for counter and uint128 max value for start`